use-cases/financial_payment_classification/financial_payment_classification.ipynb

```diff
@@ -0,0 +1,863 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    bucket_prefix = "SageMaker_Payment_Classification"
</pre><p>can we keep bucket name all lowercase?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3017/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>